src/test/crimson/seastore/test_cbjournal.cc

```diff
@@ -138,10 +138,9 @@ struct cbjournal_test_t : public seastar_test_suite_t
       epm(new ExtentPlacementManager(true)),
       cache(*epm)
   {
-    device = new nvme_device::TestMemory(CBTEST_DEFAULT_TEST_SIZE);
+    device = new nvme_device::TestMemory(CBTEST_DEFAULT_TEST_SIZE + CBTEST_DEFAULT_BLOCK_SIZE);
```

Comment:
Why do we need additional block size?

Reply:
We keep track of information about available size. Isn't it enough to check whether the journal is empty of full?

Reply:
I forgot that I removed available size variable in the header. But, I am not sure that using additional one block space here is better way. Can we add a variable in the header or in-memory variable in CBjournal? This does not require a physical block. What do you think?

Reply:
I got it.